WrightTools/cli/_units.py

```diff
@@ -0,0 +1,50 @@
+# --- import --------------------------------------------------------------------------------------
+
+
+import click
+from WrightTools import __version__ as __wt_version__
+from WrightTools.units import is_valid_conversion, get_valid_conversions, convert
+from WrightTools.exceptions import UnitsError
+
+
+# --- define --------------------------------------------------------------------------------------
+
+
+@click.command(name="convert", help="convert numbers to different units.")
+@click.version_option(__wt_version__, prog_name="WrightTools")
+@click.argument("number", type=float, nargs=1)
+@click.argument("unit", nargs=1)
+@click.argument("destination_unit", default=None, nargs=-1)
+def cli(number, unit, destination_unit=None):
+    """Convert numbers to different units."""
+
+    if int(number) == number:
+        number = int(number)
+    sig_figs = len(str(number))
+    sig_figs -= 1 if "." in str(number) else 0
+
```

Comment:
Guess we are presuming no Europeans who use commas instead of decimal points...

Reply:
Europeans have to use `.` when working with python anyways (unless they change default formatting). In any case, I don't see our notation as a meaningful hindrance.  I am fine with the syntax as is until someone raises an issue.